raven/processors.py

```diff
@@ -34,6 +34,9 @@ def process(self, data, **kwargs):
         if 'request' in data:
             self.filter_http(data['request'])
 
+        if 'extra' in data:
+            self.filter_extra(data)
```

Comment:
Actually, it ought to be data['password'] = '******' in the second example.
if you want to remove it, you can pop it too.
I'm not really happy since filter_extra and filter_http would be different. The reason is, filter_extra shouldn't interact with anything that's not extra. This is possible with the current proposal.